requirements.in

```diff
@@ -55,6 +54,6 @@ scikit-learn==1.4.1.post1  # clustering
 scipy==1.12.0  # clustering
 sentry-sdk[django]~=2.21.0
 Sphinx==1.6.3
-stripe==2.28.1
+stripe==11.5.0
```

Comment:
This breaks stripe donations as the upgraded package points to a newer API!

Reply:
good catch. I'll do a separate PR to update this and then we can go ahead with merging this pr